qiita_pet/support_files/doc/source/tutorials/getting-started.rst

```diff
@@ -1,328 +0,0 @@
-.. _getting-started:
-
-.. index:: getting-started
-
-Getting started
-===============
-
-
-.. note::
-   This is a work in progress. Please let us know of any additions or changes
-   you will like to see by emailing us at `qiita.help@gmail.com
-   <qiita.help@gmail.com>`__.
-
-If you have not yet created an account, please see the document
-:doc:`account-creation`.
-
-
-Managing your studies
----------------------
-
-Studies are the main source of data for Qiita. Studies can contain only one set
-of samples but can contain multiple sets of raw data, each of which can have a
-different preparation. Many experiments will contain only one data set, which
-includes data for all samples. Such experiments will include a single sample
-template and a single prep template.  
-
-However, Qiita can also support more complex study designs. For example
-imagine a study with 100 samples in which:
```

Comment:
The reason we don't have it in CMI workshop tutorial is that we don't use it with the test data that we provide, i.e. the idea is to show each step with example data.
I'd be happy to move it to CMI tutorial if/when we have a more complex dataset for people to work on.
In the current format, I'd rather not complicate the dataset, but we were discussing "intermediate" or "advanced" tutorial and that would fit perfectly there.

tl;dr: let's keep it in Qiita documentation for now.